src/types/contextAction/heuristicContextAction.ts

```diff
@@ -9,6 +9,7 @@ export type HeuristicContextAction =
   | 'COMMITTED_TO'
   | 'RECEIVED_AIRDROP'
   | 'GAVE_ACCESS_TO'
+  | 'REVOKED_ACCESS_FROM'
```

Comment:
@ponyjackal We shouldn't use prepositions like `from` in the actions, that should go in the sentence separately
```suggestion
  | 'REVOKED_ACCESS'
```